### PR TITLE
Change class name layout

### PR DESCRIPTION
**Pull request by fn-faisal**
Screenshot 2021-07-09 at 4 38 41 PM
Screenshot 2021-07-09 at 6 16 08 PM

### CODE REVIEW

File: src/components/CategoryList/CategoryList.scss [hunk 1]
```diff
@@ -5,130 +5,130 @@
...
-  background: #0c5987;
-  color: white;
-  border-radius: 3px;
-
-  &:hover {
-    background: #094162;
-  }
-}
\ No newline at end of file
+ @import 'stylesheets/globals.scss';
```
Comment (davimacedo) [inline src/components/CategoryList/CategoryList.scss:8]: Could you please revert these layout changes?

File: src/components/LoginForm/LoginForm.scss [hunk 2]
```diff
@@ -13,7 +13,7 @@
   left: 50%;
   margin-left: -175px;
   text-align: center;
-  width: 350px;
+  width: 300px;
```
Comment (davimacedo) [inline src/components/LoginForm/LoginForm.scss:16]: Why?

File: src/dashboard/Data/Browser/B4ABrowserToolbar.react.js [hunk 3]
```diff
@@ -74,14 +74,14 @@ let B4ABrowserToolbar = ({
...
-  let details = [];
-  if (count !== undefined) {
-    if (count === 1) {
-      details.push('1 object');
-    } else {
-      details.push(prettyNumber(count) + ' objects');
-    }
-  }
+  let details = [], lockIcon = '/lock-open-variant.svg';
+  // if (count !== undefined) {
```
Comment (davimacedo) [inline src/dashboard/Data/Browser/B4ABrowserToolbar.react.js:78]: Why did you change this logic? I believe the logic should be the same.

File: src/dashboard/Data/Browser/B4ABrowserToolbar.react.js [hunk 4]
```diff
@@ -217,9 +219,12 @@ let B4ABrowserToolbar = ({
     <Toolbar
       relation={relation}
       filters={filters}
-      section={relation ? `Relation <${relation.targetClassName}>` : `Class | ${details.join(' \u2022 ')}`}
+      details={details}
+      lockIcon={lockIcon}
+      onClickSecurity={onClickSecurity}
+      // section={relation ? `Relation <${relation.targetClassName}>` : `Class | ${details.join(' \u2022 ')}`}
```
Comment (davimacedo) [inline src/dashboard/Data/Browser/B4ABrowserToolbar.react.js:225]: Not sure if we can comment this lines. Try to click on a relation and see what happens.

**Review by davimacedo — CHANGES_REQUESTED** [review]

File: src/components/CategoryList/CategoryList.scss (hunk 1 above)
Comment (fn-faisal) [reply src/components/CategoryList/CategoryList.scss]: The changes have been reverted for the scss file.

File: src/components/LoginForm/LoginForm.scss (hunk 2 above)
Comment (fn-faisal) [reply src/components/LoginForm/LoginForm.scss]: When I added the styling to sidebar, I had to increase it's size from 300px to 350px.
Upon reverting these changes, I think I updated the LoginForm by mistake.

File: src/dashboard/Data/Browser/B4ABrowserToolbar.react.js (hunk 3 above)
Comment (fn-faisal) [reply src/dashboard/Data/Browser/B4ABrowserToolbar.react.js]: The details had the read/write permissions text and it was causing issue with the layout. I've updated the logic to pass the "read and write permissions" via another variable. I've updated the details to exclude the class permissions text and this is how the details appear now. 

Screenshot 2021-07-16 at 4 10 55 PM

File: src/dashboard/Data/Browser/B4ABrowserToolbar.react.js (hunk 4 above)
Comment (fn-faisal) [reply src/dashboard/Data/Browser/B4ABrowserToolbar.react.js]: I've reverted this.

**Review by davimacedo — APPROVED** [review]

**Pull request merged**